packages/muze-axis/src/cartesian-axis/helper.js

```diff
@@ -42,8 +42,13 @@ export const getNumberOfTicks = (availableSpace, labelDim, axis, axisInstance) =
     }
 
     numberOfValues = Math.min(numberOfTicks, Math.max(2, numberOfValues));
+    let tickValues = axis.scale().ticks(numberOfValues);
 
-    return axis.scale().ticks(numberOfValues);
+    if (numberOfValues === 2) {
```

Comment:
`2` should be defined in a constant. and That constant should be used. Enable more readablity.